lib/cache/repositories/repository.rb

```diff
@@ -12,7 +12,7 @@ def string(key)
       protected
 
       def namespace_key(key)
-        "#{@config.redis_namespace}/#{key}"
+        "#{@config.redis_namespace}/#{@config.machine_ip}/#{key}"
```

Comment:
It's the same https://github.com/splitio/ruby-client/blob/fix/include_instance_id_into_namespace/lib/splitclient-rb/split_config.rb#L57